prime-router/src/main/kotlin/Element.kt

```diff
@@ -207,6 +213,11 @@ data class Element(
                         ?: error("Invalid Code: '$formattedValue' does not match any codes for '${name}'")
                 }
             }
+            Type.POSTAL_CODE -> {
+                if (!Regex("^\\d{5}(-\\d{4})?\$").matches(formattedValue))
+                    error("Input Error: invalid postal code '$formattedValue'")
```

Comment:
Accept only 5 and 9 digit zipcodes

Reply:
What about international zip codes? - those will definitely occur.    If a file has a bad zip code, does this mean we're "failing" the whole file right now?

Reply:
Ok. It seems like the HL7 standard allows 12 letters and digits. I'll limit it to that. ZipCode is important though for our jurisdictional filter. Some districts are organized by zipcode and not county, for example.

Reply:
Yes, 12 chars seems much more general.